### PR TITLE
Update name of TPs

### PR DESCRIPTION
**Pull request by martinduke**
concurrent-streams TP is long dead

### CODE REVIEW

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -5257,11 +5257,12 @@ sequence.  A receiver is obligated to open intervening streams if a
 higher-numbered stream ID is received.  Thus, on a new connection, opening
 stream 2000001 opens 1 million streams, as required by the specification.
 
-The number of active streams is limited by the concurrent stream limit transport
-parameter, as explained in {{controlling-concurrency}}.  If chosen judiciously,
-this limit mitigates the effect of the stream commitment attack.  However,
-setting the limit too low could affect performance when applications expect to
-open large number of streams.
+The number of active streams is limited by the initial-max-streams-bidi and
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md]:
```suggestion
The number of active streams is limited by the initial_max_streams_bidi and
```

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -5257,11 +5257,12 @@ sequence.  A receiver is obligated to open intervening streams if a
...
+initial-max-streams-uni transport parameters, as explained in
```
Comment (martinthomson) [inline draft-ietf-quic-transport.md]:
```suggestion
initial_max_streams_uni transport parameters, as explained in
```

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -5257,11 +5257,12 @@ sequence.  A receiver is obligated to open intervening streams if a
...
 stream 2000001 opens 1 million streams, as required by the specification.
 
-The number of active streams is limited by the concurrent stream limit transport
-parameter, as explained in {{controlling-concurrency}}.  If chosen judiciously,
-this limit mitigates the effect of the stream commitment attack.  However,
-setting the limit too low could affect performance when applications expect to
-open large number of streams.
+The number of active streams is limited by the initial_max_streams_bidi and
+initial_max_streams_uni transport parameters, as explained in
+{{controlling-concurrency}}.  If chosen judiciously, this limit mitigates the
```
Comment (ianswett) [inline draft-ietf-quic-transport.md]:
```suggestion
{{controlling-concurrency}}.  If chosen judiciously, these limits mitigates the
```

Comment (nibanks) [reply draft-ietf-quic-transport.md]: `these limits mitigate`

File: draft-ietf-quic-transport.md [hunk 4]
```diff
@@ -5257,11 +5257,12 @@ sequence.  A receiver is obligated to open intervening streams if a
...
 stream 2000001 opens 1 million streams, as required by the specification.
 
-The number of active streams is limited by the concurrent stream limit transport
-parameter, as explained in {{controlling-concurrency}}.  If chosen judiciously,
-this limit mitigates the effect of the stream commitment attack.  However,
-setting the limit too low could affect performance when applications expect to
-open large number of streams.
+The number of active streams is limited by the initial_max_streams_bidi and
+initial_max_streams_uni transport parameters, as explained in
+{{controlling-concurrency}}.  If chosen judiciously, these limits mitigates the
```
Comment (nibanks) [inline draft-ietf-quic-transport.md]:
```suggestion
{{controlling-concurrency}}.  If chosen judiciously, these limits mitigate the
```

**Pull request merged**